resolwe_bio/processes/chip_seq/prepeak_qc.yml

```diff
@@ -149,7 +149,12 @@
         uniq -c | \
         awk '
           BEGIN{mt=0;m0=0;m1=0;m2=0} ($1==1){m1=m1+1} ($1==2){m2=m2+1} {m0=m0+1} {mt=mt+$1}
-          END{printf "%d\t%d\t%d\t%d\t%f\t%f\t%f\n",mt,m0,m1,m2,m0/mt,m1/m0,m1/m2}
+          END{ 
```

Comment:
It would be nice to have a comment explaining more than just "Get PBC bottlenecking metrics". Since this code looks quite unreadable, you can help the reader by describing what and why you did.

Reply:
Unreadable to a Pythonist :) No, I agree with you and I added some explanation.